### PR TITLE
Add bdinfo latest

### PR DESCRIPTION
**Pull request by nrlquaker**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by nrlquaker** [comment]
Something wrong with Travis:
```
Error: The 'rubocop-cask' gem is installed but couldn't find 'rubocop' in the PATH:
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/ruby/2.3.0/bin:/usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.3.3/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/Homebrew/Library/Homebrew/shims/scm
```

**Comment by commitay** [comment]
```
Error: Uh oh, could not figure out how to unpack '/Users/travis/Library/Caches/Homebrew/Cask/bdinfo--latest.dmg'
```

**Comment by nrlquaker** [comment]
For some reason when downloading from browser it downloads proper dmg, and when using curl only html with new link in content `https://www.videohelp.com/download/BDInfo%20OSX.dmg?r=NgRZLpBVmK`. The r argument changes after some time.
Any way to let curl download like its done in browser?

**Comment by commitay** [comment]
Try the additional `url` [options](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#additional-https-url-parameters)

**Comment by nrlquaker** [comment]
`referer` is working. Thanks.

File: Casks/bdinfo.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask 'bdinfo' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://www.videohelp.com/download/BDInfo%20OSX.dmg', referer: 'https://www.videohelp.com/software/BDInfo'
```
Comment (commitay) [inline Casks/bdinfo.rb]: Please move `referer:` to the next line.

https://github.com/caskroom/homebrew-cask/blob/7c99851781eea049bd4f5a77d9bf273ede41670c/Casks/bartender.rb#L10-L11

**Pull request merged**